discord-bot/src/commands/market-react-bets.ts

```diff
@@ -0,0 +1,147 @@
+import { FullMarket } from 'common/api-market-types'
+import {
+  AttachmentBuilder,
+  ChatInputCommandInteraction,
+  EmbedBuilder,
+  MessageReaction,
+  SlashCommandBuilder,
+  TextChannel,
+  User,
+} from 'discord.js'
+import { customEmojis, emojis, getBettingEmojisAsStrings } from '../emojis.js'
+import {
+  getCurrentMarketDescription,
+  getOpenBinaryMarketFromSlug,
+  getSlug,
+  handleReaction,
+} from '../helpers.js'
+import { messagesHandledViaInteraction } from '../storage.js'
+
+export const data = new SlashCommandBuilder()
+  .setName('market')
+  .setDescription('Link to a market that other users can bet on with reactions')
+  .addStringOption((option) =>
+    option
+      .setName('link')
+      .setDescription('The link to the market to bet on')
+      .setRequired(true)
+  )
+
+export async function execute(interaction: ChatInputCommandInteraction) {
+  const link = interaction.options.getString('link')
+  if (!link || !link.startsWith('https://manifold.markets/')) {
```

Comment:
Interesting that this currently is only working with the main production domain. Makes me think of whether it should be more general so it could talk to dev, a self-hosted API, etc.

Reply:
Added a domain to the config file!